### PR TITLE
Fix Wendigo BLM Droplists

### PR DESCRIPTION
**Pull request by Era-Lusiphur**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Noticed itemid 544 "Pile of Answer Sheets" dropping in Maze of Shakrami, which I knew was probably off. Researched on FFXIDB and split the Wendigo BLM droplists to correct them per FFXIDB. Adjusted the other drops into approved TH tiers that were closest to official listed droprates.

Inner Horutoto Ruins: http://ffxidb.com/zones/192/wendigo
Maze of Shakhrami: http://ffxidb.com/zones/198/wendigo

https://www.bg-wiki.com/ffxi/Treasure_Hunter

## Steps to test these changes

Import data, kill skeles.

### CODE REVIEW

**Review by Xaver-DaRed — DISMISSED** [review]

File: sql/mob_droplist.sql [hunk 1]
```diff
@@ -21915,12 +21915,16 @@ INSERT INTO `mob_droplist` VALUES (2639,0,0,1000,1031,30); -- Gusgen Chest Key (
...
--- ZoneID: 198 - Wendigo Blm
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,210);    -- Bone Chip (21.0%)
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,90);     -- Pile Of Answer Sheets (9.0%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,@RARE);     -- Pile Of Answer Sheets (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,4824,@RARE); -- Scroll Of Gravity (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (2640,2,0,1000,880,0);      -- Bone Chip (Steal)
 
+-- ZoneID: 198 - Wendigo Blm
+INSERT INTO `mob_droplist` VALUES (3238,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
```
Comment (neuromancerxi) [inline sql/mob_droplist.sql]: ID 3238 is already in use. Can you relocate this drop group to the bottom of the stack and pick the next available ID instead?

Comment (Xaver-DaRed) [reply sql/mob_droplist.sql]: imagen

Comment (Era-Lusiphur) [reply sql/mob_droplist.sql]: Ah, sorry gents. Corrected.

File: sql/mob_droplist.sql [hunk 2]
```diff
@@ -21929,12 +21929,16 @@ INSERT INTO `mob_droplist` VALUES (2639,0,0,1000,1031,30); -- Gusgen Chest Key (
...
 -- ZoneID: 192 - Wendigo Blm
--- ZoneID: 198 - Wendigo Blm
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,210);    -- Bone Chip (21.0%)
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,90);     -- Pile Of Answer Sheets (9.0%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,@RARE);     -- Pile Of Answer Sheets (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,4824,@RARE); -- Scroll Of Gravity (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (2640,2,0,1000,880,0);      -- Bone Chip (Steal)
 
+-- ZoneID: 198 - Wendigo Blm
```
Comment (neuromancerxi) [inline sql/mob_droplist.sql:21937]: Thanks for the renumber. Can you also relocate this block down to the bottom of the list (after id 3253) so the IDs stay sequential, to potentially prevent inadvertent ID reuse?

**Review by EpicTaru — CHANGES_REQUESTED** [review]
The comments in ZoneID: 198 (besides the entire block itself needing moved down to the end of the file after DropID 3237, like Neuromancer suggested) need to be aligned with the comment for the first item in the droplist (the Bone Chip (Very Common) comment).

File: sql/mob_droplist.sql [hunk 3]
```diff
@@ -21929,9 +21929,8 @@ INSERT INTO `mob_droplist` VALUES (2639,0,0,1000,1031,30); -- Gusgen Chest Key (
 INSERT INTO `mob_droplist` VALUES (2639,2,0,1000,880,0);   -- Bone Chip (Steal)
 
 -- ZoneID: 192 - Wendigo Blm
--- ZoneID: 198 - Wendigo Blm
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,210);    -- Bone Chip (21.0%)
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,90);     -- Pile Of Answer Sheets (9.0%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,@RARE);     -- Pile Of Answer Sheets (Rare, 5%)
```
Comment (EpicTaru) [inline sql/mob_droplist.sql:21933]: Comment for Pile of Answer Sheets needs shifted to align with comment of previous row (comment for Bone Chip).

**Review by EpicTaru — CHANGES_REQUESTED** [review]

File: sql/mob_droplist.sql (hunk 3 above)
Comment (TeoTwawki) [reply sql/mob_droplist.sql:21933]: you know 90% the file doesn't align right? I forget who started that. I'd rather we just have a a name field and shift all of them into it (saying the rate after is redundant, esp after aI made the `@variables`).

DSP shot down such suggestions back in the day saying "you can use a join query" as if we're going to review by running queries locally. They ain't here - just need 2 devs to approve. I'm one.

**Comment by EpicTaru** [comment]
Welcome back btw, @Era-Lusiphur, can you tell we missed ya?

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**